website/docs/integrator-section/verifiable-document/did/revoking-document.md

```diff
@@ -43,35 +43,23 @@ It is possible to revoke a signed document only if you fulfill certain **prerequ
 
 <!-- TBD v3 document sample when that releases -->
 
-- Note that the `<DEPLOYED_DOCUMENT_STORE_LOCATION>` is your `documentStore` location, it must be **deployed** on the Ethereum block chain (for now. Further implementations will relax this requirement, so that DID documents will truly be gasless).
-- Wrap your document, sign it, and append the signature to the document (follow the tutorial)
+- The `<DEPLOYED_DOCUMENT_STORE_LOCATION>` is your `documentStore` location. It must be **deployed** on the Ethereum block chain (for now. Further implementations will relax this requirement, so that DID documents will truly be gasless).
```

Comment:
As we have implemented the OCSP alternative, we can remove the line in the brackets. Also, `block chain` should be one word. 

```suggestion
- The `<DEPLOYED_DOCUMENT_STORE_LOCATION>` is your `documentStore` location. It must be **deployed** on the Ethereum blockchain. 
```

Reply:
added your suggestion